draft-ietf-tls-esni.md

```diff
@@ -787,6 +815,48 @@ where HKDF-Expand-Label and HKDF-Extract are as defined in {{RFC8446}}. The
 value of ServerHello.random[0:24] is generated as usual by invoking a secure
 random number generator (see {{RFC8446}}, Section 4.1.2).
 
+When confirming ECH acceptance, the backend server SHOULD reply to the
+"handshake_padding" extension solicited by the client, computing the padding
+length as described in {{server-padding}}.
```

Comment:
I'd recommend pulling this out into a separate issue entirely. The client-side padding policy took a while to get consensus, so trying to spell the server-side story here might delay landing of the general mechanism. Can we do that and replace this with an open issue?

```suggestion
[[OPEN ISSUE: provide server-side padding policy based on Certificate, CertificateVerify(?), ALPN list, and
possibly other (future) parameters.]]
```

Reply:
I think it's fine to RECOMMEND something that is obsoleted by a better idea later on. But as I mentioned above, if this change is likely to be controversial, then we should take it to the list. In addition, I think we should consider publishing -08 beforehand. There's already substantial changes.

Reply:
Yeah, I expect this to be controversial. I'd take it to the list.